public/app/plugins/datasource/jaeger/responseTransform.ts

```diff
@@ -68,62 +60,6 @@ function toSpanRow(span: Span, processes: Record<string, TraceProcess>): TraceSp
   };
 }
 
-export function createTableFrame(data: TraceResponse[], instanceSettings: DataSourceInstanceSettings): DataFrame {
```

Comment:
Code search shows this is not used.

The only results are related to tempo datasource, no with imports from jaeger

<img width="803" height="374" alt="Image" src="https://github.com/user-attachments/assets/3b24192a-666b-4e9b-9ea7-80772e5d36f1" />